docs/topics/rn-new-features.adoc

```diff
@@ -7,66 +7,22 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.Updated {WebName} interface
+.MTA plugin for IntelliJ IDEA
 
-The current release contains an updated {WebName} interface with improved project management features and additional transformation targets.
+The current release supports the {ProductName} plugin for the IntelliJ IDEA IDE. The plugin provides assistance directly in IntelliJ IDEA for developers making changes for a migration or modernization effort. It has the same functionality as other {ProductName} plugins: it analyzes your projects using customizable rulesets, marks migration issues in the source code, and provides guidance to fix issues.
 
-.{ProductName} Operator for {ocp-short}
+The plugin can be installed through link:https://plugins.jetbrains.com/marketplace[the IntelliJ marketplace].
 
-The current release contains the {ProductName} Community Operator for installing the {WebName} on {ocp-short} 4.6 and later versions.
+.Rules for migrating from JBoss EAP 7.3 to Boss EAP 7.4
 
-.Installation in multiple namespaces on {ocp-short}
+The current release contains rules for migrating from JBoss EAP 7.3 t o Boss EAP 7.4. The target of these rules is `eap:7`.
 
-The current release can be installed in multiple namespaces on {ocp-short}. Previously, the {WebName} could be installed in only one namespace.
+.Rules for migrating from JBoss EAP XP 2 to JBoss EAP XP 3
```

Comment:
I would remove the 2nd "JBoss EAP XP". It's clear enough if your write "JBoss EAP XP 2 to 3".

Reply:
Same for JBoss EAP 7.3 ... to 7.4 (above)